tests/unit/src/test/scala/org/scalafmt/tests/CoverageSuite.scala

```diff
@@ -0,0 +1,253 @@
+  check("def f(x: A = 1): B")         // Term.Param (default parameter)
+  check("s\"a $b\"")                  // Term.Interpolate
+  check("<a>b {c}</a>")               // Term.Xml
+
+  // meta.Pkg
+  checkSource("package a")
+  checkSourceStructural(
+    """|package a.b {
+     |  package c {
+     |
+     |  }
+     |}""".stripMargin
+  )
+  checkSourceStructural(
+    """|package a {
+     |  class A
+     |}
+     |class B""".stripMargin
+  )
+  checkTreeSource(Pkg(Term.Name("a"), List())) // no parent
+  checkSource("package object a") // Pkg.Object
+
+  // meta.{Import, Importee, Importer}
+  check("import a.b")
+  check("import a.b, c.d")
+  check("import a._, c._")
+  check("import a._")
+  check("import a.{ b, c }")
+  check("import a.{ b => c }")
+  check("import a.{ b => _ }")
+
+  // meta.Self
+  check("trait A { self: B => }")
+  check("trait A { _: B => }")
+  check("trait A { self => }")
+  checkStructural("trait A { this: B => }")
+
+  // meta.Template
+  check("new A {}")
+  check("class A extends B with C with D")
+  check("class Y extends { val a = 1 } with X")
+  check("new { val a = 1 } with A {}")
+
+  // meta.Member
+  
+  // meta.Decl
+  check("val a: Int")      // Decl.Val
+  check("var b: Long")     // Decl.Var
+  check("def f: String")   // Decl.Def
+  check("type T")          // Decl.Type
+
+  // meta.Defn
+  check("val a = 1")              // Defn.Val
+  check("var a = 1")              // Defn.Var
+  check("var a: Int = _")         // Defn.Var
+  check("def a = 1")              // Defn.Def
+  check("def f = macro m")        // Defn.Macro
+  check("type T = Int")           // Defn.Type
+  check("class A")                // Defn.Class
+  check("class A(b: B)")          // Defn.Class
+  check("class A private (b: B)") // Defn.Class
+  check("trait A")                // Defn.Trait
+  check("object A")               // Defn.Object
+
+  // meta.Ctor
+  check("class A { def this(a: A) = this() }") // Ctor.Secondary
+  check(
+    """|class A {
+     |  def this(a: A) = {
+     |    this()
+     |
+     |    f(b)
+     |  }
+     |}""".stripMargin
+  ) // Ctor.Secondary
+
+  // meta.Mod
+  check("@tailrec def f = 1")       // Mod.Annotation
+  check("private[foo] val a = 1")   // Mod.Private
+  check("protected[foo] val a = 1") // Mod.Protected
+  check("implicit val a = 1")       // Mod.Implicit
+  check("final val a = 1")          // Mod.Final
+  check("sealed trait a")           // Mod.Sealed
+  check("override def f = 1")       // Mod.Override
+  check("case object B")            // Mod.Case
+  check("abstract class A")         // Mod.Abstract
+  check("class A[+ T]")             // Mod.Covariant
+  check("class A[- T]")             // Mod.Contravariant
+  check("lazy val a = 1")           // Mod.Lazy
+  check("class A(val b: B)")        // Mod.ValParam
+  check("class A(var b: B)")        // Mod.VarParam
+  check("inline def f = 1", dotty)  // Mod.Inline
+
+  // meta.Pat
+  checkCase("case `1` =>")              // Lit
```

Comment:
Might be worth to add similar tests for patterns on the lhs of Defn.Val

```scala
val 1 = a
val a.b = c
...
```

Reply:
Yup, the focus of this PR is to get 100% Coverage. We can do this later.